keyboards/dz60/keymaps/bingocaller/README.md

```diff
@@ -0,0 +1,114 @@
+# MacOS standard 60% keymap with Vim-like arrows
```

Comment:
Could you also rename this file to be lowercase? Windows is case-insensitive, so for consistency and to avoid problems all filenmames should be in lowercase :)

Reply:
don't disagree, but convention on github is usually this file is all caps.

Reply:
I reject GitHub's reality and substitute my own.

Reply:
Sure, I can do that! :)
Though isn't it a pretty common convention to use uppercase for readme-files, especially in Git repos?